steps/perfecto-upload/0.0.6/step.yml

```diff
@@ -0,0 +1,68 @@
+title: Perfecto Upload
+summary: |
+  Upload APK/IPA files to Perfecto cloud repository
+description: |
+  Upload your build files to Perfecto for remote device testing. Upload your IPA/APK file to Perfecto repository
+website: https://github.com/PerfectoCode/Bitrise_Perfecto_FileUpload
+source_code_url: https://github.com/PerfectoCode/Bitrise_Perfecto_FileUpload
+support_url: https://github.com/PerfectoCode/Bitrise_Perfecto_FileUpload/issues
+published_at: 2020-06-27T19:13:46.420922+05:30
+source:
+  git: https://sreevatsa-ramya:Sreesmrithi123%24@github.com/PerfectoCode/Bitrise_Perfecto_FileUpload.git
```

Comment:
Credentials shouldn't be there.

Reply:
It is possible that it was specified in this format for the checked out git repository, and got inserted in this format.
Will check if it is possible to filter out from Bitrise tooling side. Until now pleas check the git remote URL in your local checked out repo.